voms-mapfile-default

```diff
@@ -55,3 +55,4 @@
 "/lhcb/Role=production/Capability=NULL" lhcbprod
 "/lhcb/*" lhcb
 "/clas12/*" clas12
+
```

Comment:
Looks like there's a stray whitespace change here

Reply:
Yep I noticed it fixed